src/ildasm/dis.cpp

```diff
@@ -315,7 +315,6 @@ BOOL enumEHInfo(const COR_ILMETHOD_SECT_EH* eh, IMDInternalImport *pImport, DWOR
     COR_ILMETHOD_SECT_EH_CLAUSE_FAT ehBuff;
     const COR_ILMETHOD_SECT_EH_CLAUSE_FAT* ehInfo;
     BOOL    fTryInCode = FALSE;
-    if(g_ehInfo) VDELETE(g_ehInfo);
```

Comment:
Found the problem, submitting a new patch and reverting this line to the original.